nixos/modules/config/update-users-groups.pl

```diff
@@ -226,7 +226,7 @@ sub parseUser {
     if ($u->{createHome}) {
         make_path($u->{home}, { mode => 0700 }) if ! -e $u->{home} and ! $is_dry;
```

Comment:
why not use the correct mode here as well? This way you don't set it twice

Reply:
Note that this is so today. I only changed line 229. It may be that the author didn't like setting the permission before setting the correct uid and gid. I didn't change because I do not know if there is any other good reason for doing so.

Reply:
I've changed the code to directly set the permissions as you suggest.